core/src/main/scala/com/nvidia/spark/rapids/tool/profiling/ProfileOutputWriter.scala

```diff
@@ -199,4 +210,41 @@ object ProfileOutputWriter {
 
     sb.toString()
   }
+
+  /**
+   * Format an input string as pretty JSON
+   */
+  def formatJsonString(json: String, indentSize: Int = 2): String = {
```

Comment:
was this copied from somewhere?  I would think there is a scala library that does this. what about json4s?  or com.fasterxml.jackson.core.util.PrettyPrinter?

Reply:
Thanks @tgravescs! Updated this to use `json4s`.